Drop-In/src/main/java/com/braintreepayments/api/DropInResult.java

```diff
@@ -27,46 +26,46 @@
      */
     public static final String EXTRA_ERROR = "com.braintreepayments.api.dropin.EXTRA_ERROR";
 
-    static final String LAST_USED_PAYMENT_METHOD_TYPE =
-            "com.braintreepayments.api.dropin.LAST_USED_PAYMENT_METHOD_TYPE";
-
     private String deviceData;
     private String paymentDescription;
 
-    private DropInPaymentMethodType paymentMethodType;
+    private DropInPaymentMethod paymentMethodType;
     private PaymentMethodNonce paymentMethodNonce;
 
     DropInResult() {}
 
-    void setPaymentMethodNonce(@Nullable PaymentMethodNonce paymentMethodNonce) {
-        setPaymentMethodNonce(paymentMethodNonce, new PaymentMethodNonceInspector());
+    DropInResult setPaymentMethodNonce(@Nullable PaymentMethodNonce paymentMethodNonce) {
+        return setPaymentMethodNonce(paymentMethodNonce, new PaymentMethodInspector());
     }
 
     @VisibleForTesting
-    void setPaymentMethodNonce(@Nullable PaymentMethodNonce paymentMethodNonce, PaymentMethodNonceInspector nonceInspector) {
+    DropInResult setPaymentMethodNonce(@Nullable PaymentMethodNonce paymentMethodNonce, PaymentMethodInspector nonceInspector) {
```

Comment:
I think there were some rebase issues with this class - we removed the builder pattern here in #307

Reply:
Yeah good call. The rebase had some conflicts I'll update.